data/campaigns/Descent_Into_Darkness/scenarios/08_Alone_at_Last.cfg

```diff
@@ -1080,10 +1083,52 @@
 
         {MOVE_UNIT (id=Malin Keshar) 1 8}
 
-        [message]
-            speaker=Malin Keshar
-            message= _ "The terrain is hazardous in these mountains. Neither the paladins nor Dela will be able to pursue me."
-        [/message]
+        # GitHub Issue #7005: Handle presence/absence of the factions/leaders
+        [if]
+            # Phase one: Before Cadaeus or Dela have arrived
+            # Unlikely, but handle the case for completeness.
+            [variable]
+                name=cadaeus_arrived
+                boolean_equals=no
+            [/variable]
+            [and]
+                [variable]
+                    name=dela_arrived
+                    boolean_equals=no
+                [/variable]
+            [/and]
+            [then]
+                [message]
+                    speaker=Malin Keshar
+                    message= _ "The terrain is hazardous in these mountains. No one should be able to pursue me."
+                [/message]
+            [/then]
+
+            # Phase two: After Cadaeus arrives but before Dela arrives
+            # Even if Cadaeus' forces are wiped out at this point it seems reasonable to infer there could be more paladins 'off screen'.
+            [elseif]
+                [variable]
+                    name=dela_arrived
+                    boolean_equals=no
+                [/variable]
+                [then]
+                    [message]
+                        speaker=Malin Keshar
+                        message= _ "The terrain is hazardous in these mountains. The paladins will not be able to pursue me."
+                    [/message]
+                [/then]
+            [/elseif]
+
+            # Phase three: Cadaeus and Dela have arrived
+            # Generalise as 'Dela's men' in case Dela herself is killed.
+            # The scenario of both Cadaeus and Dela being killed is handled in the 'alternate victory' event.
+            [else]
+                [message]
+                    speaker=Malin Keshar
+                    message= _ "The terrain is hazardous in these mountains. Neither the paladins nor Dela’s men will be able to pursue me."
```

Comment:
Change, or add a `# po:` comment to say that `men` should be translated as gender-neutral?